webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/reports/ExecutionBatch.scala

```diff
@@ -166,6 +174,7 @@ final case class NoUserRulesDefined(
  */
 final case class NoReportInInterval(
     expectedConfig: NodeExpectedReports
+  , expirationDateTime: DateTime
```

Comment:
it's because we discover that it has no report in interval at time T, and we need to ensure at T+expire that it's still the case (and if so, serialize it)